scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -564,16 +589,148 @@ def validate_model_assets(latest_asset_config: assets.AssetConfig, validated_ass
                     f"But its status is {online_deployment_status}. "
                     "Please ensure that that online inference is validated for the model."
                 )
-                return 1
+                error_count += 1
 
-        return 0
+        return error_count
     except Exception as e:
         logger.log_error(
             f"Exception when confirming validation results for model {latest_asset_config.name}. Exception {e}"
         )
         return 1
 
 
+def validate_model_scenario(
+    asset_file_name_with_path: Path,
+    model: Model,
+    min_sku_prop_name: str,
+    recommended_skus_prop_name: str,
+    compute_allowlist_tags_name: str,
+) -> bool:

```

Comment:
Should this function be returning a bool (to count this as one check) or an int (error count)?

Reply:
Done